cfme/tests/services/test_catalog_item.py

```diff
@@ -134,10 +134,8 @@ def test_add_button(catalog_item, appliance):
     """
     button_name = catalog_item.add_button()
     view = appliance.browser.create_view(BaseLoggedInPage)
-    if appliance.version.is_in_series('5.8'):
-        message = 'Button "{}" was added'.format(button_name)
-    else:
-        message = 'Custom Button "{}" was added'.format(button_name)
+
+    message = f'Custom Button "{button_name}" was added'
```

Comment:
I like @mshriver idea but you need to major changes in model page also taken care about navigation as catalog level and automate buttons are different.

Optional: (hot fix)
As  `add_button` method accepting `**kwargs`
```
btn_data = {
    "text": gen_numeric_string(start="btn_"),
    "hover": gen_numeric_string(15, start="btn_hvr_"),
    "image": "fa-user",
}
catalog_item.add_button(**btn_data)
view = appliance.browser.create_view(BaseLoggedInPage)
message = f'Custom Button "{btn_data["hover"]}" was added'
```